packages/hoprd/src/index.ts

```diff
@@ -180,7 +180,8 @@ async function main() {
     if (argv.healthCheck) {
       const http = require('http')
       const service = require('restana')()
-      service.get('/healthcheck/v1/version', (_, res) => res.send(FULL_VERSION))
+      const version = require('../package.json')
```

Comment:
This is wrong. FULL_VERSION is correct.

Reply:
Typescript doesn't know FULL_VERSION so it most likely does not exist

Reply:
It's in constants. There is a missing import.   ` const version = require('../package.json') is definitely wrong.`